types/src/write_set.rs

```diff
@@ -217,57 +183,60 @@ impl WriteOp {
         use WriteOp::*;
 
         match self {
-            Creation { data, .. } | Modification { data, .. } => data.len(),
-            Deletion { .. } => 0,
+            Creation(v) | Modification(v) => v.bytes().len(),
+            Deletion(..) => 0,
```

Comment:
Can we document that `Returns the size of the write op. If write op is a deletion, 0 is returned.`? Otherwise it is not clear that `op.size()` is not the same as `op.bytes().unwrap().len()` (technically you cannot even call `len` here)

Reply:
Actually they are the same? (which makes it more confusing. I'm gonna change the name of the method)